### PR TITLE
Added Contact Section with all necessary styling

### PR DESCRIPTION
**Pull request by chawkimbarki**
# Made changes :
- added contact section according to today's activity
-  styled according to the Figma template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -140,6 +140,23 @@ <h3 class="ability-title">Skills</h3>
...
+                    <input type="text" maxlength="30" required>
+                </div>
+                <div class="input-wrapper" id="email-input">
+                    <label for="email">Your Email</label>
+                    <input type="email" name="email" id="email" required>
+                </div>
+                <textarea required maxlength="500" name="message" id="message" cols="30" rows="10" placeholder="Enter your message here..."></textarea>
+                <button type="submit" id="contact-button" class="primary-button">Start Collaboration</button>
+            </form>
+        </section>
```
Comment (MaganaAsiati) [inline index.html:159]: - [ ] You did a great job in the design of your contact form it looks nice however you forget to implement the part i shows in the picture below kindly implement it as required in the project your design must stick to the Figma design as much as possible :+1: 
image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @jossi30, I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] [OPTIONAL] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by chawkimbarki** [comment]
Hello @MaganaAsiati
thanks for the detailed review, but I think that this is not a part of the contact section
image

as you can see here, they only asked us to do the contact form, not the social links part
image

please if I miss understood the activity explain it to me if possible
I really appreciate it.

Thanks in advance

File: index.html [hunk 2]
```diff
@@ -140,6 +140,36 @@ <h3 class="ability-title">Skills</h3>
...
+                <div class="input-wrapper" id="name-input">
+                    <label for="name">Your Name</label>
+                    <input type="text" maxlength="30" required>
+                </div>
+                <div class="input-wrapper" id="email-input">
+                    <label for="email">Your Email</label>
+                    <input type="email" name="email" id="email" required>
+                </div>
+                <textarea required maxlength="500" name="message" id="message" cols="30" rows="10" placeholder="Enter your message here..."></textarea>
+                <button type="submit" id="contact-button" class="primary-button">Start Collaboration</button>
```
Comment (TuyishimireEric) [inline index.html:157]: - [ ] [OPTIONAL] Kindly remember to add button interactions such as **hover, active (when it's pressed), disabled, and enable (default state)** on all your buttons as shown in the Figma template. 😂 
image

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @chawkimbarki     👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️ Professional Pull Request 👍
✔️ No Linter errors 👍   

## Optional Requirements 
_Check the comments below_
 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**